### ISSUE TITLE
Letter Fragments

### ISSUE THREAD

**Issue by unisys12** [Bungie-net/api#426]
We in the Destiny Lore community have always had to pull a lot of our info from the API and if not for the API you guys and gals provide, our community would be way smaller than it is. With that said, we have one area that I think everyone would love access to - Letter Fragments. 

My query this morning returns roughly 40 of them. They can easily be found in the manifest through the itemCategoryHashes property with a value of "3747263590.0". Which, for whatever reason has a displayName of "Story Prizes". Anyway! To my point. 

When a player turns in one of these Letter Fragments, say to Cayde-6 for example, you get an awesome display of text that tells a small part of his storied past. This can not be found anywhere in the API.

The Lore community would love access to text that is displayed after turning in one of these letter fragments and allow our community to discern the most information that they can from them in the most efficient manner.  The team at Ishtar has had a ton of help from the community with transcribing as much of the game as possible. we in turn have made those transcriptions searchable and digestible for research. Adding these Letter Fragments to a WishList would be a blessing to our community. 

Thank you and your team for all that you do! Happy Hunting!

**Comment by vthornheart-bng** [comment]
Hmm, interesting!  I'll have to dig around and see where that's at!  Thanks for reporting this!

**Comment by vthornheart-bng** [comment]
TFS 615854

**Comment by lowlines** [comment]
I've currently got a page set up with the hand-typed lore blurbs.
https://lowlidev.com.au/destiny/caydes-journal

Apparently there are only 5 letter fragments obtainable in the game, the rest are either placeholders or obtained in a way that no one has discovered yet.

Here's my json file with the lore text.
https://lowlidev.com.au/destiny/d2/caydes-journal.json

Here are all the hashes. I was using the inventory.stackUniqueLabel property to group them by destination as there always appears to be 4 per destination with the current known Letter Fragments occupying the first index (ie story_prize_planetx0)

```
2243093675
2283361089
2413899256
2511194732
2706486842
2888153229
3210853326
3406569464
3406569465
3406569466
3406569467
3638113959
3752885090
95988838
140100789
405406988
405406989
405406990
405406991
622896628
622896629
622896630
622896631
682132521
1131875567
1273792320
1273792321
1273792322
1273792323
1398781283
1602487168
1827816196
1383325066
1383325067
1383325064
1383325065
```

**Comment by vthornheart-bng** [comment]
Good digging as always lowlines!  Thanks!  #hired

**Comment by unisys12** [comment]
Yeah, thanks @lowlines. I should've included the hashes in my original post. :(

**Comment by vthornheart-bng** [comment]
No worries!  I'd have dug them up either way, all good!
